src/auth/computeclient.ts

```diff
@@ -144,9 +144,9 @@ export class Compute extends OAuth2Client {
   protected async getTrustBoundaryUrl(): Promise<string> {
     const email = await this.resolveServiceAccountEmail();
     const trustBoundaryUrl = SERVICE_ACCOUNT_LOOKUP_ENDPOINT.replace(
-      '{service_account_email}',
-      encodeURIComponent(email),
-    );
+      '{universe_domain}',
```

Comment:
I am leaving this comment here just to be sure that we are skipping trust boundary lookup if the universe is not googleapis.com. Is this the case?